src/cloud/project/project-upgrade.md

```diff
@@ -88,23 +88,23 @@ To update the `.magento.app.yaml` file:
 
 ## Upgrading the Magento application
 
-Review the [Magento technology stack requirements]({{site.baseurl}}/guides/v2.3/install-gde/system-requirements-tech.html) before upgrading your Magento application.
+Review the [Magento technology stack requirements] before upgrading your Magento application.
 
 ### Back up the database
 
 {% include cloud/backup-db.md %}
 
 ### Complete the upgrade
 
-If you use PHP version 7.2, you must remove the `mcrypt` extension from the [`extensions` section of the `.magento.app.yaml` file]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options). For Pro projects, you need to create a support ticket to completely disable the `mcrypt` extension.
+If you use PHP version 7.2, you must remove the `mcrypt` extension from the [extensions section of the .magento.app.yaml file]. For Pro projects, you need to create a support ticket to completely disable the `mcrypt` extension.
 
  {:.bs-callout-info}
-When upgrading to 2.3 from 2.2, you must verify that the `composer.json` file contains `"Zend\\Mvc\\Controller\\": "setup/src/Zend/Mvc/Controller/"` in the `"psr-4":` section of the `autoload` property.
+When upgrading to 2.3.X from 2.2.X, you must verify that the `composer.json` file contains `"Zend\\Mvc\\Controller\\": "setup/src/Zend/Mvc/Controller/"` in the `"psr-4":` section of the `autoload` property.
```

Comment:
I noticed in some places the x is corrected to and from lowercase. Just pointing out. I don't have a particular preference.